package.json

```diff
@@ -65,7 +65,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12",
     "@testing-library/react-hooks": "^7",
-    "@testing-library/user-event": "^12",
+    "@testing-library/user-event": "^14",
```

Comment:
It would be nice if we can do this upgrade in a different PR to limit the scope of this PR.